matrix.json

```diff
@@ -0,0 +1,77 @@
+{
+    "uuid": "4BA68789-FF92-4649-B7D8-65BCE3966589",
+    "version": "2.0",
+    "organisation": {
+        "name": "Shift CryptoAG",
+        "web": "https://shiftcrypto.ch/",
+        "email": "support@shiftcrypto.ch",
+        "location": "Switzerland",
+        "established": "2018-01-23"
+    },
+    "wallet": {
+        "name": "Bitbox02",
+        "web": "https://shiftcrypto.ch/bitbox02/",
+        "socials": [
+            {
+                "type": "twitter",
+                "slug": "shiftcryptoHQ"
+            },
+            {
+                "type": "medium",
```

Comment:
Removed medium. No there is no "blog" entry. The "socials" array is designed to take a pre-defined list of "available" 3rd party platforms ala: Twitter,Discord,Gitter,Medium,Linkedin,Facebook,Telegram,Reddit,Youtube,Instagram, just so that we can easily abstract the logic to determine the URL pattern from a given slug/handle.

Added a complete list of socials from the website (apart from YouTube - the format seems to have changed since last I looked)

Reply:
> > I think there is a general issue here in that walletmatrix seems to mix hardware wallet and software wallet features. Related discussion: [bitcoin-dot-org/Bitcoin.org#3509](https://github.com/bitcoin-dot-org/Bitcoin.org/issues/3509)
> 
> This is my point.

Understood. The short and sweet answer is that for _hardware_ wallets specifically; then companion _mobile_ and/or _desktop_ wallets and their features are not counted.

As for "applets" that allow a coin to be managed directly within the H/W device itself, that are manufactured, engineered and supplied by the _same_ manufacturer/developer as the device, then these _should_ be counted.

I own a Ledger Nano S, and it has an "ETH" and "BTC" app. From memory (it was some years ago) the "BTC" app/applet came with it, but I had to install the "ETH" one - I could be wrong, it was a long time ago. Regardless, they were produced by Ledger themselves and as far as WalletMatrix is concerned, can be included.

Note that featured currencies/coins are _utterly incidental_ to WalletMatrix, so BitBox02 can list a subset of them, and that will be fine. If Shift Crypto want to maintain _all_ of them within matrix.json going forward, then we're happy for that to happen too :-)

Reply:
> Is attestation mentioned somewhere aka the feature that companion wallets can verify the bitbox is real?

Can you describe this feature in greater detail? In terms of _verification_ then at the moment searches for Android wallets are run through the WalletScrutiny (https://walletscrutiny.com) service - but not H/W wallet firmware at present (I note that WS has labelled BitBox02 as "Reproducible" so I have just added this feature to the WM backlog!).

Reply:
1) ledger nano S and X come with no applets at all (at least the ones I bought last year)
2) okay when it's about the intrinsic features of the HWW itself obviously some things like coin control etc would need to be marked as "doesnt apply" or something like that, and the coin/token list is gonna get LONG if all gets included, personally I would be in favor of only listing "direct" coins and make a mention that these types of tokens are also supported like "ERC20 tokens").
3) regarding the attestation feature it's not the reprod firmware that is its own thing. Attestation means that the HWW has some special keys/certificates in the secure element built into them at manufacturing, that can be passed a challenge for attestation that is then signed (to prove owning the private key) and the application can put the certificate used against a list of known ones and when they match you can be pretty sure your bitbox is real, if the attestation fails (e.g. because the bitbox is fake, or some weird interaction bugs or by nature of having a developer device) you will get something like this when trying to unlock.
![image](https://user-images.githubusercontent.com/6696524/134462851-0f597417-7684-477f-9221-930fec22fa9b.png)
the ledger in fact has a similar feature when you connect it to ledger live.

Reply:
1). You're probably right - but TBH the list of coins is incidental (and limited within WalletMatrix - I will publish a list of supported ones (that will probably help figure out which ones you can include anayway!)
2). Let me know if anything needs to be corrected with an inline comment (apart from "coin_control" which I've noted)
3). Wicked, thanks for the detail. I have added it to our draft v4 schema document